.github/workflows/ci.yml

```diff
@@ -218,7 +218,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, ubuntu-24.04-arm, macos-13, windows-2019, windows-2022]
-        go-version: ["1.22.10", "1.23.4"]
+        go-version: ["1.23.6", "1.24.0"]
```

Comment:
there is 1.24.1 and 1.23.7 now. https://go.dev/doc/devel/release#go1.24.minor

should we just use the latest direclty?

Reply:
That change is not yet present on the main and release branches. 

I will make that bump on those(main.2.0,1.7) branches and then update this PR.